app/views/embed/iframe.html.erb

```diff
@@ -5,6 +5,15 @@
     <script>
       <%= 'if (window.jQuery) { jQuery.fx.off = true; }' if Rails.env.test? %>
     </script>
+    <script src="//js.honeybadger.io/v6.5/honeybadger.min.js" type="text/javascript"></script>
```

Comment:
What does the `//` mean here? I'm unfamiliar with this syntax.

Reply:
That means, use whatever protocol delivered this page.